### DISCUSSION TITLE
Ability to use emojis

### DISCUSSION THREAD

**Discussion by markuslindmark** [six7/figma-tokens#472] [Ideas]
I'm using emoji's to indicate light and dark theme. It useful because it do not take up too much space as text would. (very smal right panel in Figma when working with color styles). The problem is that emojis get transformed to glyphs in the jsonfile when syncing with GitHub. 

For example white flag 🏳️ gets transformed to the glyph ð³ï¸ 

Is there any easy fix for that?

**Comment by six7** [comment]
This is mostly related to this issue: https://github.com/six7/figma-tokens/issues/388

No fix yet, but hopefully soon! 🙏
